react/components/ProductImages/components/InnerZoom.js

```diff
@@ -28,22 +28,45 @@ export default class InnerZoom extends Component {
     },
   }
 
+  constructor(props){
+    super(props)
+    this.imageZoomed = React.createRef()
+    this.contentDiv = React.createRef()
+  }
+
+  getScale() {
+    let scale = {
+      x : 1,
+      y : 1
+    }
+
+    if(this.contentDiv.current && this.imageZoomed.current) {
+      let { offsetWidth, offsetHeight } = this.contentDiv.current
```

Comment:
Why are you using let variables instead of const ones if the values do not change?